maintainers/maintainer-list.nix

```diff
@@ -7451,6 +7451,12 @@
     githubId = 1538622;
     name = "Michael Reilly";
   };
+  onixie = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add onixie`` and sort it alphabetically.